topics/icrc_X_multiple_canister_call.md

```diff
@@ -0,0 +1,237 @@
+# ICRC-X: Call batch canisters
+
+[![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](#)
+[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./icrc_25_signer_interaction_standard.md)
+[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--49-ffcc222.svg)](./icrc_49_call_canister.md)
+
+<!-- TOC -->
+
+- [ICRC-X: Call batch canisters](#icrc-x-call-batch-canisters)
+  - [Summary](#summary)
+  - [Processing](#processing)
+    - [Examples](#examples)
+      - [Successful Batch Call](#successful-batch-call)
+      - [Batch Call with Error](#batch-call-with-error)
+
+## Summary
+
+This Method can be used by the relying party to request batch call to 3rd party canister executed by the signer using the requested identity. In order to prevent misuse of this method all `icrcX_call_batch_canisters` requests are subject to user approval.
```

Comment:
```suggestion
This method can be used by the relying party to request a batch call to 3rd party canister executed by the signer using the requested identity. In order to prevent misuse of this method all `icrcX_call_batch_canisters` requests are subject to user approval.
```